dotcom-rendering/src/web/layouts/FrontLayout.tsx

```diff
@@ -60,65 +64,6 @@ const isToggleable = (
 	} else return index != 0 && !isNavList(collection);
 };
 
-const getMerchHighPosition = (
```

Comment:
We're moving this code from `FrontsLayout` to `getAdPositions` so that we can unit test it.

Reply:
You can unit test component files I think by exporting `getMerchHighPosition` and creating FrontLayout.test.ts, but I think its new location makes sense anyway :)